node.js

```diff
@@ -80,6 +80,9 @@ module.exports = {
       files: ['*.ts', '*.tsx'],
       parser: '@typescript-eslint/parser',
       plugins: ['@typescript-eslint/eslint-plugin'],
+      parserOptions: {
+        project: './tsconfig.json'
+      },
```

Comment:
This seems like something we should document in our Readme and release notes, as I can imagine some rare edge-cases where someone has their tsconfig in different location